src/core/effects/qgsgloweffect.cpp

```diff
@@ -33,6 +33,8 @@ QgsGlowEffect::QgsGlowEffect( const QgsGlowEffect &other )
   , mSpreadUnit( other.spreadUnit() )
   , mSpreadMapUnitScale( other.spreadMapUnitScale() )
   , mBlurLevel( other.blurLevel() )
+  , mBlurUnit( other.mBlurUnit )
+  , mBlurMapUnitScale( other.mBlurMapUnitScale )
```

Comment:
Likely fixes https://github.com/qgis/QGIS/issues/34089

Reply:
No, it doesn't

Reply:
Maybe because it's missing in the operator= method?

Reply:
They were indeed missing in operator=, but fixing it doesn't fix the issue. Perhaps a pure GUI issue

Reply:
Yep - gui issue. I'll fix after this is merged.

Reply:
> I'll fix after this is merged.

I'm on it. Fix under way

Reply:
Fixed by https://github.com/qgis/QGIS/pull/34135/commits/01412a7bec32d736d13a6166f3ecbc2f5230284c . The fix is only effective if combined with the copy constructor/operator= fix . So this whole PR should probably be backported